### ISSUE TITLE
Image fabric A11y Pass

### ISSUE THREAD

**Issue by awalia13** [OfficeDev/office-ui-fabric-react#8233]
Tracking all of the issues brought up with the 03/06 accessibility pass on Image fabric control
Details on individual bugs can be found on [Sharepoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7B4de95463-5ae6-4ac4-a046-ee613df0e0fc%7D&action=default&uid=%7B4DE95463-5AE6-4AC4-A046-EE613DF0E0FC%7D&ListItemId=1862&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble)
- [ ] Bug1: [Accessibility][Usable]: Screen does not auto scroll whenever scan mode focus moves to the image on the page.
- [ ] Bug2: [Accessibility][Usable][Suggestion]: Instances of complex infographics/charts could be provided for testing purpose.
- [ ] Bug3: [Accessibility][Keyboard Navigation]: All the scroller on the page is not accessible by keyboard.

**Comment by dzearing** [comment]
I see no actionable bugs here. The `Image` component is a glorified wrapper around `img` so I expect really nothing interactive here.
